src/governance.cpp

```diff
@@ -982,16 +965,42 @@ bool CGovernanceObject::IsValidLocally(const CBlockIndex* pindex, std::string& s
 
     // IF ABSOLUTE NO COUNT (NO-YES VALID VOTES) IS MORE THAN 10% OF THE NETWORK MASTERNODES, OBJ IS INVALID
 
-    if(GetAbsoluteNoCount(VOTE_SIGNAL_VALID) > mnodeman.CountEnabled(MSG_GOVERNANCE_PEER_PROTO_VERSION)/10){
-         strError = "Automated removal";
-         return false;
+    if(GetAbsoluteNoCount(VOTE_SIGNAL_VALID) > mnodeman.CountEnabled(MSG_GOVERNANCE_PEER_PROTO_VERSION)/10) {
+        strError = "Automated removal";
+        return false;
     }
 
     // CHECK COLLATERAL IF REQUIRED (HIGH CPU USAGE)
 
-    if(fCheckCollateral){
-        int nConf = 0;
-        if(!IsCollateralValid(nCollateralHash, GetHash(), strError, nConf, GOVERNANCE_FEE_TX)){
+    if(fCheckCollateral) {
+        if(nObjectType == GOVERNANCE_OBJECT_TRIGGER) {
+            CMasternode mn;
+            bool mnfound = mnodeman.Get(pubkeyMasternode, mn);
```

Comment:
Either naming e.g. `fFound` or this could really be a part of the following `if` at line 980 since `mnfound` is not used anywhere else.